web/src/stream_popover.js

```diff
@@ -566,9 +570,57 @@ export async function build_move_topic_to_stream_popover(
         event.stopPropagation();
     }
 
+    function localeSafeCaseInsensitiveInlucde(topics, current_topic) {
+        return topics.some( t => t.localeCompare(current_topic, undefined, { sensitivity: 'accent' }) === 0);
+    }
```

Comment:
I think this is pretty overkill and probably not worth it. Often times readable code is better than complex since it saves time for maintainers to go over your pull requests, as well as anyone else coming in to work on this later on.

Looking online, it seems like `localeCompare` also has poor performances.